rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.health/20_request_timeout.yml

```diff
@@ -17,8 +17,8 @@
   - is_true:   timed_out
   - gte:       { number_of_nodes:           1 }
   - gte:       { number_of_data_nodes:      1 }
-  - match:     { active_primary_shards:     0 }
-  - match:     { active_shards:             0 }
+  - gte:       { active_primary_shards:     0 }
```

Comment:
This test only cares about reaching a timeout and not how many shards are there.
In some cases it can be 0 and in some 1, depends if `.security` index is created before test executes.